runtime/src/bank.rs

```diff
@@ -7158,6 +7085,32 @@ impl Bank {
             reload,
         )
     }
+
+    pub fn withdraw(&self, pubkey: &Pubkey, lamports: u64) -> Result<()> {
```

Comment:
didn't know we have now `#[cfg(feature = "dev-context-only-utils")]`-ed `impl Bank {...}`. :)